sdv/multi_table/base.py

```diff
@@ -119,15 +119,12 @@ def __init__(self, metadata, locales=['en_US'], synthesizer_kwargs=None):
         self._fitted_sdv_version = None
         self._fitted_sdv_enterprise_version = None
         self._synthesizer_id = generate_synthesizer_id(self)
-        SYNTHESIZER_LOGGER.info(
-            '\nInstance:\n'
-            '  Timestamp: %s\n'
-            '  Synthesizer class name: %s\n'
-            '  Synthesizer id: %s',
-            datetime.datetime.now(),
-            self.__class__.__name__,
-            self._synthesizer_id
-        )
+        SYNTHESIZER_LOGGER.info({
+            'EVENT': 'Instance',
+            'TIMESTAMP': datetime.datetime.now(),
+            'SYNTHESIZER CLASS NAME': self.__class__.__name__,
+            'SYNTHESIZER ID': self._synthesizer_id
+        })
```

Comment:
🤔 what if we use other handler that is not `CSV`?  

CC: @amontanez24 
Are we going to support multiple handlers or just `csv` would be the end goal of this ? If so I think we can simplify things a bit regarding the `logger`.

Reply:
We can always write other `logging.Formatter` classes that take in this input dict and output it however we want it formatted.

Reply:
I think csv will be it